### PR TITLE
Dala 351 cleanup frontend auth

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request Cleanup Frontend Auth
Cleans up unused parts of the frontend
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] ~At least one E2E Test exists testing the new feature~
- [x] Documentation is updated as required
- [ ] ~The automated deployment is updated if required~
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [ ] ~The new feature is manually used/tested/observed on dev server~
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] ~There is at least one picture for each story, which was created before coding has started~
- [ ] ~If any work on the UI is to be merged, those changes were also documented in the Figma~ 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-csvconverter/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM alpine:latest
+FROM ubuntu:latest
 
-COPY ./CompanyInformationWithEuTaxonomyData.json /storedData/CompanyInformationWithEuTaxonomyData.json
+RUN apt-get update && apt-get upgrade -y && apt-get install samba && apt-get clean
 
-CMD cp /storedData/*.json /testData
+CMD ["tail","-f","/dev/null"]
```
Comment (MarcMDfine) [inline dataland-csvconverter/Dockerfile:5]: Gehören diese Änderungen mit in den Cleanup frontend auth PR?

File: dataland-frontend/src/utils/SearchTaxonomyPageCompanyDataRequester.ts [hunk 2]
```diff
@@ -63,15 +62,11 @@ export async function getCompanyDataForTaxonomyPage(
     | "Hdax"
     | "Dax50Esg",
   onlyCompanyNames: boolean,
-  getKeycloakInitPromise: any,
-  keycloak_init: any
+  getKeycloakInitPromise: any
 ): Promise<Array<object>> {
```
Comment (MarcMDfine) [inline dataland-frontend/src/utils/SearchTaxonomyPageCompanyDataRequester.ts:66]: Sollte das hier nicht keycloakInitPromise : Promise<Keycloak> sein? Wird ja meines Erachtens keine Funktion, sondern der Promise übergeben.

File: dataland-frontend/src/App.vue [hunk 3]
```diff
@@ -42,12 +42,10 @@ export default {
...
         this.keycloak_init_promise_finished = false;
       }
-      return this.keycloak_init_promise;
     },
   },
   provide() {
     return {
-      keycloak_init: this.keycloak_init,
       getKeycloakInitPromise: () => {
         return this.keycloak_init_promise;
```
[the remark this answers is not in the document]
Comment (ppuetsch) [reply dataland-frontend/src/App.vue:50]: renamed

File: dataland-frontend/src/services/ApiClients.ts [hunk 4]
```diff
@@ -7,11 +7,9 @@ import {
 } from "@/../build/clients/backend/api";
 export class ApiClientProvider {
   keycloak_init_promise: any;
-  keycloak_init: any;
 
-  constructor(keycloak_init_promise: any, keycloak_init: any) {
+  constructor(keycloak_init_promise: any) {
     this.keycloak_init_promise = keycloak_init_promise;
-    this.keycloak_init = keycloak_init;
```
[the remark this answers is not in the document]
Comment (ppuetsch) [reply dataland-frontend/src/services/ApiClients.ts:14]: renamed

File: dataland-frontend/src/services/ApiClients.ts [hunk 5]
```diff
@@ -7,11 +7,9 @@ import {
 } from "@/../build/clients/backend/api";
 export class ApiClientProvider {
   keycloak_init_promise: any;
-  keycloak_init: any;
 
-  constructor(keycloak_init_promise: any, keycloak_init: any) {
+  constructor(keycloak_init_promise: any) {
```
[the remark this answers is not in the document]
Comment (ppuetsch) [reply dataland-frontend/src/services/ApiClients.ts:11]: fixed

File: dataland-frontend/src/utils/SearchTaxonomyPageCompanyDataRequester.ts (hunk 2 above)
Comment (ppuetsch) [reply dataland-frontend/src/utils/SearchTaxonomyPageCompanyDataRequester.ts:66]: fixed

**Pull request merged**